src/helpers/helper.ts

```diff
@@ -0,0 +1,20 @@
+import { createNewGame } from "redux/store/gameReducer";
+
+export function createGame() {
```

Comment:
better would be if u add  expect length as parametr in function
`export function createGame(length?:number = 2) `